tests/tests/test-stake.ts

```diff
@@ -18,7 +18,7 @@ describeWithMoonbeam("Moonbeam RPC (Stake)", `simple-specs.json`, (context) => {
   });
 
   step("issuance minted to the sole validator for authoring blocks", async function () {
-    const issuanceEveryRound = 49n * GLMR;
+    const issuanceEveryRound = 0n * GLMR;
```

Comment:
Oh, never mind, need to increase the number of blocks to wait because the new `BlocksPerRound` is 600 and not 20....

Reply:
Getting this error when I change the  createAndFinalize block loops to 1200 and 1800 from 40 and 60 respectively.
```
 Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/4meta5/ong/ps/moonbeam/tests/tests/test-stake.ts)
```

@joelamouche  do you know if there is a way to simulate this much block time passing without triggering this error. If not, I will remove the test.